.eslintignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+.git
+.gitignore
+.idea
```

Comment:
Please, don't forget about empty lines at the end of the files and `.gitignore` (I remember that you were going to fix this in your later PRs, just a reminder)